### ISSUE TITLE
View Single Experiments from the Database in Online Viewer

### ISSUE THREAD

**Issue by ZiDa20** [ZiDa20/Biophysical_Essentials#173]
If the experiment is already in the database - it would be nice to have a viewing option for single experiments instead of always starting a new offline analysis ?!

**Comment by mouzkolit** [comment]
@ZiDa20  can we not just implement this into the online analysis, load from file or load from database?

**Comment by ZiDa20** [comment]
yep ..that was the intention of this issue: a button in online analysis opening a popup showing the database entries ... with some filter and search functions .. and maybe a checkbox column for multi-select
